downstream/assemblies/hub/assembly-container-user-access.adoc

```diff
@@ -9,17 +9,17 @@ ifdef::context[:parent-context: {context}]
 :context: configuring-user-access-containers
 
 [role="_abstract"]
-To determine who can access and manage images in your {PlatformNameShort}, you must configure user access for container repositories in your {PrivateHubName}.
+To determine who can access and manage {ExecEnvShort}s in your {PlatformNameShort}, you must configure user access for container repositories in your {PrivateHubName}.
 
 include::hub/ref-container-permissions.adoc[leveloffset=+1]
 include::hub/proc-create-groups.adoc[leveloffset=+1]
-include::hub/proc-assigning-permissions.adoc[leveloffset=+1]
+// [hherbly]: proc-assigning-permissions seems to repeat proc-create-groups.adoc include::hub/proc-assigning-permissions.adoc[leveloffset=+1]
 
 .Additional resources
 
-* See <<container-registry-group-permissions, Container registry group permissions>> to learn more about specific permissions.
+* See <<container-registry-group-permissions, Container registry team permissions>> to learn more about specific permissions.
```

Comment:
Is this supposed to be a link?

Reply:
I'm just going to remove this. It points to the module right before this one, which doesn't give any more "specific" information than what the user has just read. Thanks for catching it!